src/components/profession-hero/ProfessionHero.css

```diff
@@ -0,0 +1,222 @@
+.profession {
+    display: flex;
+    flex-direction: column;
+    padding: 95px 0 100px 0;
```

Comment:
Check this padding on mobile widths, the title and subtitle expand to the edge of the page, there should be some padding there.

Reply:
i looked, but i'm not seeing what youre talking about. The section itself has no left/right padding, yes, but the title group has a width of 90%, so the title and subtitle never touch the edge of the page, at least not on my computer. or maybe youre talking about something else?

Reply:
I can't see what I was talking about either, that must have been a mistake on my end, zooming into the screen or something. My bad Colin! This all looks great.